docker/android/Dockerfile

```diff
@@ -1,4 +1,7 @@
-FROM ubuntu:zesty
+# sys_image can be overridden from the CLI
+ARG sys_image=debian:jessie
```

Comment:
Having this first makes `python build.py --interactive` fail. Building directly on `FROM debian:jessie` works (docker 1.12.3).